kernel/api/src/main/java/org/sakaiproject/util/api/ValueEncryptionService.java

```diff
@@ -0,0 +1,21 @@
+package org.sakaiproject.util.api;
+
+public interface ValueEncryptionService {
```

Comment:
So, we should remove the interface and invoke directly the implementation?
We don't think it is a problem to allow different implementations...